File: StockList Demo for tvOS/StockListCell.m
//
//  StockListCell.m
//  StockList Demo for tvOS
//
//  Copyright © 2016 Lightstreamer. All rights reserved.
//

#import "StockListCell.h"


@implementation StockListCell


#pragma mark -
#pragma mark Initialization

- (id) initWithCoder:(NSCoder *)decoder {
    if (self = [super initWithCoder:decoder]) {
        
        // Nothing to do, actually
    }
    
    return self;
}


#pragma mark -
#pragma mark Properties

@synthesize nameLabel= _nameLabel;
@synthesize lastLabel= _lastLabel;
@synthesize timeLabel= _timeLabel;
@synthesize dirImage= _dirImage;
@synthesize changeLabel= _changeLabel;
@synthesize bidLabel= _bidLabel;
@synthesize askLabel= _askLabel;
@synthesize minLabel= _minLabel;
@synthesize maxLabel= _maxLabel;
@synthesize refLabel= _refLabel;
@synthesize openLabel= _openLabel;


@end

File: StockList Demo for tvOS/StockListCell.swift
//  Converted to Swift 5.4 by Swiftify v5.4.25812 - https://swiftify.com/
//
//  StockListCell.swift
//  StockList Demo for tvOS
//
//  Copyright © 2016 Lightstreamer. All rights reserved.
//

import UIKit

class StockListCell: UITableViewCell {
    // MARK: -
    // MARK: Properties
    @IBOutlet private(set) weak var nameLabel: UILabel?
    @IBOutlet private(set) weak var lastLabel: UILabel?
    @IBOutlet private(set) weak var timeLabel: UILabel?
    @IBOutlet private(set) weak var dirImage: UIImageView?
    @IBOutlet private(set) weak var changeLabel: UILabel?
    @IBOutlet private(set) weak var bidLabel: UILabel?
    @IBOutlet private(set) weak var askLabel: UILabel?
    @IBOutlet private(set) weak var minLabel: UILabel?
    @IBOutlet private(set) weak var maxLabel: UILabel?
    @IBOutlet private(set) weak var refLabel: UILabel?
    @IBOutlet private(set) weak var openLabel: UILabel?

    // MARK: -
    // MARK: Initialization

    required init?(coder decoder: NSCoder) {
        super.init(coder: decoder)
            // Nothing to do, actually
    }

    // MARK: -
    // MARK: Properties
}
